src/pages/About/AboutTheExperts/Councils/NortheastCoverCropsExperts.js

```diff
@@ -0,0 +1,60 @@
+/*
+  This file contains the HelpComponent, helper functions, and styles
+  The HelpComponent is a static  help page that has FAQ, how to use, data dictionary, and information sheets
+  RenderContent contains all the text listed in the about section
+  styled using CustomStyles from ../../shared/constants
+*/
+
+import { Typography } from '@mui/material';
+import React from 'react';
+
+const NortheastCoverCropExperts = () => {
```

Comment:
can you have one component for mapping over the experts and then stick the arrays of experts in constants.js?